components/camel-google-pubsub/pom.xml

```diff
@@ -47,83 +47,7 @@
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-pubsub</artifactId>
             <version>${google-cloud-pubsub-version}</version>
-            <exclusions>
```

Comment:
Are you sure about removing the exclusions?

Reply:
PubSub 1.108.1 has a dependency on GRPC 1.57.1 and it doesn't work with the GRPC 1.28.0 as defined in the parent.
Should I increase the version in the parent, add a pubsub specific version in the parent of leave it like so?

Reply:
Do you mean 1.27.1? I think it makes sense to add a property placeholder only for pubsub and use it in this component only. We can do the same for the karaf feature. Camel-grpc is based on 1.28.0 so we cannot generally downgrade

Reply:
Sorry, GRPC 1.30.2. So that would mean a general upgrade. Would that be possible?

Reply:
I don't think camel-grpc is ready for this. So I believe it would be better to use a specific version only in this component

Reply:
I've created an explicit dependency on GRPC 1.30.2 now.